<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BossWidget</class>
 <widget class="QWidget" name="BossWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BossWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="eternalCorridorBox">
     <property name="title">
      <string>Eternal Corridor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="eternalCorridorCharacteristicLayout" stretch="1,1">
        <item>
         <widget class="QGroupBox" name="eternalCorridorElementBox">
          <property name="title">
           <string>Element</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="formAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <item row="0" column="0">
            <widget class="QCheckBox" name="eternalCorridorElementRandom">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Random</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="eternalCorridorSpecialMagicBox">
          <property name="title">
           <string>Special / Magic</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QCheckBox" name="eternalCorridorSpecialMagicRandom">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Random</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="eternalCorridorAppearanceBox">
        <property name="title">
         <string>Appearance</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
         <item>
          <widget class="QCheckBox" name="eternalCorridorRandomNewAppearance">
           <property name="text">
            <string>Random New Appearance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="eternalCorridorAppearanceTextureBox">
           <property name="title">
            <string>Texture</string>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="1" column="0">
             <widget class="QCheckBox" name="eternalCorridorAppearanceTextureRandom">
              <property name="text">
               <string>Random</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="eternalCorridorAppearanceTextureRandomColor">
              <property name="text">
               <string>Random Color</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="eternalCorridorAppearanceTextureIncludeCompatible">
              <property name="text">
               <string>+Include Compatible</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="storyBox">
     <property name="title">
      <string>Story</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="storyCharacteristicLayout" stretch="1,1">
        <item>
         <widget class="QGroupBox" name="storyElementBox">
          <property name="title">
           <string>Element</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item row="0" column="0">
            <widget class="QCheckBox" name="storyElementRandomAll">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Random All</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="storyElementRandomElemental">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Random Elemental</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="storySpecialMagicBox">
          <property name="title">
           <string>Special / Magic</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="storySpecialMagicRandom">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Random</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="storyAppearanceBox">
        <property name="title">
         <string>Appearance</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="storyAppearanceTextureBox">
           <property name="title">
            <string>Texture</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="storyAppearanceTextureColorBasedOnElement">
              <property name="text">
               <string>Color Based On Element</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="storyAppearanceTextureRandomColor">
              <property name="text">
               <string>Random Color</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
